service/resource/deployment/deployment.go

```diff
@@ -0,0 +1,44 @@
+package deployment
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/azuretpr"
+
+	"github.com/giantswarm/azure-operator/service/key"
+)
+
+const (
+	clusterIDParameterName     = "clusterID"
+	clusterSetupDeploymentName = "cluster-setup"
+	clusterSetupTemplate       = "cluster_setup.json"
+	templateBaseURI            = "https://raw.githubusercontent.com/giantswarm/azure-operator/master/service/arm_templates"
```

Comment:
For now we use the template from master on GitHub. We can iterate on this and decide if we use the commit SHA or blob storage.